packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewSummary.tsx

```diff
@@ -69,7 +69,7 @@ const Headline = styled.div`
     font-size: 16px;
     font-weight: 600;
     margin-top: 20px;
-    word-break: break-all;
+    word-break: normal;
```

Comment:
`break-word` would fit better.
Some tokens can have 18 decimals and amount can be also huge
<img width="323" alt="Screenshot 2024-09-08 at 8 17 51" src="https://github.com/user-attachments/assets/b010349e-5402-4f33-9eb0-94641c4cfa1e">